github/repo_overrides.yml

```diff
@@ -4,6 +4,15 @@ alphagov/smartanswers:
 alphagov/govuk-coronavirus-content:
   allow_squash_merge: true
 
+alphagov/govuk-coronavirus-vulnerable-people-form:
+  up_to_date_branches: true
+
+alphagov/govuk-coronavirus-business-volunteer-form:
+  up_to_date_branches: true
+
+alphagov/govuk-coronavirus-find-support:
+  up_to_date_branches: true
```

Comment:
Not one for this PR, but it strikes me as a little odd for us to define repo-specific overrides in this generalised repo. I'd have expected some sort of `.govuk_saas_config.yml` convention in downstream repos instead 🤔 

I guess it makes it easier to add/remove/rename properties, but is another one of those things you just need to know exists or else it all works like magic. I think I'll add a PR to mention this in https://docs.publishing.service.gov.uk/manual/architecture-deep-dive.html, as I hadn't quite made the connection.

Reply:
I suppose it's easier if you have it in one place rather than the script having to scan every repo for a config file that someone may delete because they don't know what it does. 🤔

Reply:
Here's the PR I've been meaning to raise: https://github.com/alphagov/govuk-developer-docs/pull/2569